pkg/util/testingjobs/pod/wrappers.go

```diff
@@ -62,6 +62,34 @@ func MakePod(name, ns string) *PodWrapper {
 	}}
 }
 
+func MakeOCPPod(name, ns string) *PodWrapper {
```

Comment:
Can we add a comment explaining why we need one OCP Pod? and can't use `MakePod`